src/short_id.rs

```diff
@@ -20,14 +20,21 @@ use bitcoin::{BlockHash, Txid};
 
 #[derive(Copy, Clone, Debug, Display, PartialEq, Eq)]
 #[display(Debug)]
```

Comment:
we need here
```suggestion
#[display(doc_comments)]
```

since now we have meaningful comments

Reply:
Great thanks for pointing that out :raised_hands: